MediaManager/Plugin.MediaManager.Abstractions/IMediaFile.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System;
 using System.ComponentModel;
-using Plugin.MediaManager.Abstractions.Implementations;
 
 namespace Plugin.MediaManager.Abstractions
 {
+    /// <summary>
+    /// Information about the mediafile
+    /// </summary>
+    /// <seealso cref="System.ComponentModel.INotifyPropertyChanged" />
     public interface IMediaFile : INotifyPropertyChanged
     {
-        Guid Id { get; set; }
-
         /// <summary>
         /// Indicator for player which type of file it should play
         /// </summary>
-        MediaFileType Type { get; set; }
+        Implementations.MediaFileType Type { get; set; }
```

Comment:
I would prefer if you used: using Plugin.MediaManager.Abstractions.Implementations instead of Implementations.MediaFileType